src/nandini/BankingSystem.java

```diff
@@ -0,0 +1,70 @@
+package nandini;
+
+public class BankingSystem {
+	
+	int debitCount;
+	int creditCount;
+	int currentBalance;
+	static int debitCounttotal;
+	static int creditCounttotal;
+	static int totalbalanceCount;
+	static int printTotalbalanceCount;
```

Comment:
There is one requirement related this as well for individual user, that you missed here.
i want individual printTotalBalanace count as well.